### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by sahlminkok**
### In this project we:
1. Add my project list to the HTML file
2. Add about me to the HTML file
3. Style the above HTML file with CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,5 +50,155 @@ <h2 class="Description1">I'am a Software Developer</h2>
...
+    <li>HTML</li>
+    <li>Bootstrap</li>
+    <li>Ruby</li>
+  </ul>
+  <a href="#" class="button">See project</a>
+  </div>
+  <div class="project2">
+    <h3>Profesional Art <br>Printing Data</h3>
+  <p>
+    A daily selection of privately personalized reads; no accounts or sign-ups
```
Comment (christianbiring1) [inline index.html:94]: - [ ] Kindly make sure your project list is visible, as you may notice on the UI, Only the `see project` button can be seen for now. Please ensure the user can see the project for a better user experience.
### Your design :point_down: 
image
### Figma template :point_down: 
image

File: style.css [hunk 2]
```diff
@@ -102,3 +91,308 @@ p {
   align-items: center;
   margin: 0% 0% 0% 7%;
 }
+
+.headline-tagline {
+  height: 519px;
+  background-image: url("images/Group.png");
```
Comment (christianbiring1) [inline style.css]: - [ ] YOu may consider using the correct path for your image.
```suggestion
  background-image: url("./images/Group.png");
```

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @sahlminkok , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sahlminkok) [reply index.html:94]: @christianbiring1 I have committed the changes

Comment (christianbiring1) [reply index.html:94]: @sahlminkok , you can go ahead and submit again, a reviewer will review the changes.

Good luck!

File: style.css [hunk 3]
```diff
@@ -102,3 +91,308 @@ p {
...
+  display: grid;
+  grid-template-rows: repeat(6, auto);
+  border: 0;
+  gap: 24px;
+}
+
+.project2 {
+  display: flex;
+  flex-direction: column;
+  background-image: url(images/placeholder.png);
```
Comment (amon-cofie) [inline style.css]: You made a great attempt to fix the changes that the previous reviewer requested 👍 . However, the problem still persists. I did some digging however and I realized that the image you are trying to access is being spelled wrongly. It starts with a capital letter and hence this line should look something like this: 

```
background-image: url(images/Placeholder.png);
```

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @sahlminkok , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

🚀 Good commit messages
🚀 Descriptive pull request
🚀 No linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @sahlminkok 👋

Great job implementing the changes suggested by the reviewer 💪

**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks professional, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**